tmd/Topology/tests/test_topology_analysis.py

```diff
@@ -64,3 +65,133 @@ def test_distance_horizontal():
 def test_distance_horizontal_unnormed():
     nt.ok_(analysis.distance_horizontal_unnormed(sample_data_0, sample_data_0)==0.0)
     nt.ok_(analysis.distance_horizontal_unnormed(sample_data_0, sample_data_1, bins=4) == 1.0)
+
+
+def test_closest_ph():
+
+    random_values = np.random.random(4).tolist()
+
+    ph_list = [
+                [
+                    [16.90, 6.68],
+                    [10.52, 5.98],
+                    [74.11, 0.00]
+                ],
+                [
+                    [3.01, 1.58],
+                    [15.22, 0.18],
+                    [60.48, 0.00]
+                ],
+                [
+                    [9.78, 5.30],
+                    [7.66, 1.60],
+                    [24.00, 0.00]
+                ],
+                [
+                    [6.05, 2.01],
+                    [3.91, 1.41],
+                    [8.05, 0.00]
+                ],
+                [
+                    [2.78, 0.87],
+                    [6.12, 0.21],
+                    [21.24, 0.00]
+                ],
+                [
+                    [4.99, 4.06],
+                    [4.38, 2.92],
+                    [6.79, 0.00]
+                ],
+                [
+                    [4.99, 4.06],
+                    [4.38, 2.92],
+                    [4.79, 0.00]
+                ]
+    ]
+
+    for ph in ph_list:
+        for bar in ph:
+            bar += random_values
+
+    target_extent = 6.0
+
+    closest_ph, closest_index = \
+    analysis.closest_ph(ph_list, target_extent, method='from_above', return_index=True)
+
+    assert closest_index == 5, (closest_index, 5)
```

Comment:
minor: assert_equal print a nicer output when the assert fails

Reply:
From `nose`? `numpy.testing` has only for arrays, no?

Reply:
yes, the one from nose

Reply:
`numpy.testing` has `assert_equal` as well!